client/my-sites/backup/rewind-flow/steps/missing-credentials.tsx

```diff
@@ -1,20 +1,22 @@
 import { Button, Gridicon } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
 import { FunctionComponent } from 'react';
-import { backupPath, settingsPath } from 'calypso/lib/jetpack/paths';
+import { settingsPath } from 'calypso/lib/jetpack/paths';
 import { useDispatch } from 'calypso/state';
 import { recordTracksEvent } from 'calypso/state/analytics/actions/record';
 
 interface Props {
 	siteSlug: string | null;
 	enterCredentialsEventName: string;
 	goBackEventName: string;
+	goBackUrl: string;
```

Comment:
Adding support for the `goBackUrl` so we could specify where to redirect the customer if the customer is in a regular restore or granular restore flow.